src/prop_current/cbsd_antenna_gain.py

```diff
@@ -0,0 +1,24 @@
+'''
```

Comment:
Can you move this file to harness/reference_models/{$folder_name} https://docs.google.com/document/d/17ZvP0Ju8eENJ4jRKx0c2WL7yCwX22J8xjaGA93Cc_FI/edit#heading=h.ul0kthrnnhpb